### PR TITLE
frontEnd 2nd PR

### PR DESCRIPTION
**Pull request by Dazeofcitywalk**
**Summary:**
change frontend return style from "1. summary" to "1. link: summary:"

**Details:**
· delete previous style 
· add new style
· pass the external test and delete test case

**Screenshot:**
截屏2023-08-22 下午3 39 32

**Others:**
There is still a path problem in importing the backend modules. I couldn't call the `get_5_summary_from_5_articles` function to present outputs without importing those modules.
So I used external test case to make sure the update output style is working (presented in `test.py` from the 1st commit, then deleted it from the 2nd commit).
If the importing problem is solved, I think it will give the return we want.

### CODE REVIEW

File: Phraends.py [hunk 1]
```diff
@@ -45,12 +45,22 @@
         save_search_history(ticker)
 
         # Run the function that generates key phrases into a list (this will be replaced by model function eventually)
-        key_phrases = phrases_output_example()
+        #key_phrases = phrases_output_example()
+
+        # new: get link and summary
+        return_2list = Phraends_Flask.get_5_summary_from_5_articles()
```
Comment (YLee2121) [inline Phraends.py]: change to 

```python
links, summaries = Phraends_Flask.get_5_summary_from_5_articles()
```

File: Phraends.py [hunk 2]
```diff
@@ -45,12 +45,22 @@
...
-        st.write("Key phrases for the stock ticker you entered:")
-        for i, sentence in enumerate(key_phrases, start=1):
-            st.write(f"{i}. {sentence}")
+        #st.write("Key phrases for the stock ticker you entered:")
+        #for i, sentence in enumerate(key_phrases, start=1):
+        #    st.write(f"{i}. {sentence}")
+
+        # new: Output key phrases onto website
+        st.write("Key links and summaries for the stock ticker you entered:")
+        for i in range(len(return_2list[0])):
```
Comment (YLee2121) [inline Phraends.py]: change to 

```python
for link, summary in zip(links, summaries):
...
...
```

**Review by YLee2121 — CHANGES_REQUESTED** [review]
tiny change required. 

please also revise after my comment.

**Comment by alex308248** [comment]
Wondering why couldn't call the `get_5_summary_from_5_articles`. Did you try something like the following?
api_returns = Phraends_Flask.get_5_summary_from_5_articles(ticker)

File: Phraends.py (hunk 1 above)
Comment (YLee2121) [reply Phraends.py]: Also, you need to import this function.

```python
from Phraends_Flask import get_5_summary_from_5_articles
```

**Review by YLee2121 — APPROVED** [review]
@Weirdojord 
Crawler has bug when I use "MMM" as ticker

**Pull request merged**